e2e/src/api/specs/asset.e2e-spec.ts

```diff
@@ -242,6 +247,73 @@ describe('/asset', () => {
       });
     });
 
+    it('should get the asset faces', async () => {
+      const response = await request(app)
+        .put('/system-config')
+        .set('Authorization', `Bearer ${admin.accessToken}`)
+        .send({
+          ...(await getSystemConfig(admin.accessToken)),
+          metadata: { faces: { import: true }, tags: { import: true } },
+        });
```

Comment:
You can use the SDK for everything except the ending being tested.

Reply:
Fixed by a07d425fb